public/js/reducers/user.js

```diff
@@ -45,18 +46,30 @@ export default function user(state, action) {
     });
   }
 
-  if (action.type === actionTypes.LOADING_USER_SUBSCRIPTIONS) {
+  if (action.type === actionTypes.LOADING_USER_SUBS) {
     return Object.assign({}, initialUserState, state, {
       subscriptions: initialUserState.subscriptions,
     });
   }
 
-  if (action.type === actionTypes.GOT_USER_SUBSCRIPTIONS) {
+  if (action.type === actionTypes.GOT_USER_SUBS) {
     return Object.assign({}, initialUserState, state, {
       subscriptions: action.subscriptions,
     });
   }
 
+  if (action.type === actionTypes.GOT_FILTERED_USER_SUBS) {
+    return Object.assign({}, initialUserState, state, {
+      // This is short-lived data only relevant to the current
+      // operation.
```

Comment:
again, I think the scope of the current operation is the current management view